### PR TITLE
Migrate all release notes

### PR DESCRIPTION
**Pull request by roadlittledawn**
## Summary of changes
Changes max pages for all content types _except_ basic pages. The content brought in is mostly release notes. Additionally, I discovered that the migration API was excluding several troubleshooting docs. I updated a setting in Drupal and it brought more of those files in.

Also, in order to troubleshoot corrupt image file, I ensured I was using 2.11.1 of `gatsby-plugin-sharp` which then gave me a useful error message identifying the file.

## Notes
Basic pages contain a lot of nested code blocks which need to be de-indented manually, so we'll need to address that in later MMF. There were additional issues with collapsers (in super nested docs like enabling AWS Lambda) that we'll need to address as well. We have the content files we need to do rest of this MMF work.

## Why the build failed initially
When migrating this content, the build failed due to:
* a known issue with the `kafka-monitoring-integration.mdx` where indented codeblocks must be manually de-indented
* a corrupt .png file (events.png) in `src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/images`. not 100% sure of cause but in Drupal there were 2 images with same name (one named `events.png` and the other `Events.png`). I reuploaded both images and renamed them to unique names. Did that in Drupal prod and dev

### CODE REVIEW

**Review by zstix — DISMISSED** [review]

**Comment by roadlittledawn** [comment]
looks like the build had the same error i was experiencing. some digging so far, it sounds like a broken source image.

**Comment by roadlittledawn** [comment]
this image file looks suspicious. https://github.com/newrelic/docs-website/blob/campfire/migrate-most-agent-stuff/src/content/whats-new/2020/11/images/%25E2%2580%25A2-New-Relic-ONE%25E2%2584%25A2-copy.png

**Comment by roadlittledawn** [comment]
made some changes to .png files that had periods or escape chars in their filename. but my build still fails locally with same error as that amplify build

**Comment by roadlittledawn** [comment]
Found the culprit. After ensuring I was on 2.11.1 of `gatsby-plugin-sharp` I ran build and got useful error pointing me to the corrupt image: https://github.com/newrelic/docs-website/blob/campfire/migrate-most-agent-stuff/src/content/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/images/events.png

It now builds successfully. Will have to see if this happens for every migration and if so, if I can solve it at the source in Drupal

**Review by zstix — APPROVED** [review]
Looks good, and it's building for me!

File: src/content/docs/browser/new-relic-browser/getting-started/TEST-PAGE-Browser.mdx [hunk 1]
```diff
@@ -1,41 +0,0 @@
----
```
Comment (jerelmiller) [inline src/content/docs/browser/new-relic-browser/getting-started/TEST-PAGE-Browser.mdx:1]: This is the page that @mmfred added. Let's keep this file around.

**Review by jerelmiller — CHANGES_REQUESTED** [review]
Spotted the one file that we should try and keep around for @mmfred 's usability test. If there are nav links that got removed, we should also make sure they are kept.

File: src/content/docs/browser/new-relic-browser/getting-started/TEST-PAGE-Browser.mdx (hunk 1 above)
Comment (roadlittledawn) [reply src/content/docs/browser/new-relic-browser/getting-started/TEST-PAGE-Browser.mdx:1]: you read my mind. was going to hold on merging this until i can sync drupal prod to dev and run migration again as part of this PR

Comment (jerelmiller) [reply src/content/docs/browser/new-relic-browser/getting-started/TEST-PAGE-Browser.mdx:1]: Sweet! Thank you!!

**Review by jerelmiller — APPROVED** [review]

**Pull request merged**